lib/iris/analysis/__init__.py

```diff
@@ -1026,18 +1026,35 @@ def _percentile(data, axis, percent, **kwargs):
     If a new additive dimension is formed, then it will always be the last
     dimension of the resulting percentile data payload.
 
+    Kwargs:
+
+    * percentile_method (string) :
```

Comment:
Afternoon. My affiliation is Met Office.
I have based the code on v1.13 as that is the version we are currently using. It is up-to-date with that branch, so a rebase does nothing. I can move it to v2 if that is desirable?

I have changed the comment and moved the keyword argument into the interface as suggested.